### PR TITLE
Add integration test to verify that TbV download was successful

### PR DESCRIPTION
**Pull request by senselessdev1**
## PR Summary
Verifies that all expected files are present.

## Testing

In order to ensure this PR works as intended, it is:

* [ ] unit tested.
* [ ] other or not applicable (*additional detail/rationale required*)

## Compliance with Standards

As the author, I certify that this PR conforms to the following standards:

* [ ] Code changes conform to [PEP8](https://www.python.org/dev/peps/pep-0008) and docstrings conform to the Google Python [style guide](https://google.github.io/styleguide/pyguide.html?showone=Comments#38-comments-and-docstrings).
* [ ] A well-written summary explains what was done and why it was done.
* [ ] The PR is adequately tested and the testing details and links to external results are included.

### CODE REVIEW

File: integration_tests/verify_tbv_download.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+from av2.map.map_api import ArgoverseStaticMap
+
+
+logger = logging.getLogger(__name__)
+
+MIN_NUM_SWEEPS_PER_LOG: Final[int] = 40
+MIN_NUM_IMAGES_PER_CAMERA: Final[int] = 80
+
+# every lane segment should have 10 keys only.
+EXPECTED_LANE_SEGMENT_ATTRIB_KEYS = [
```
Comment (benjaminrwilson) [inline integration_tests/verify_tbv_download.py:25]:
```suggestion
EXPECTED_LANE_SEGMENT_ATTRIB_KEYS: Final[Tuple[str, ...]] = (
```

File: integration_tests/verify_tbv_download.py [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    "lane_type",
+    "left_lane_boundary",
+    "left_lane_mark_type",
+    "right_lane_boundary",
+    "right_lane_mark_type",
+    "successors",
+    "predecessors",
+    "right_neighbor_id",
+    "left_neighbor_id",
+]
```
Comment (benjaminrwilson) [inline integration_tests/verify_tbv_download.py:37]:
```suggestion
)
```

File: integration_tests/verify_tbv_download.py [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+        "width_px",
+    ]
+
+    # intrinsics should be provided for each camera
+    for camera_enum in list(RingCameras):
+        assert camera_enum.value in intrinsics_df["sensor_name"].tolist()
+
+    verify_log_map(data_root=data_root, log_id=log_id)
+
+    # new_log_id = f"{log_uuid}__{season}_{year}"
```
Comment (benjaminrwilson) [inline integration_tests/verify_tbv_download.py:117]: Is this supposed to be here?

File: integration_tests/verify_tbv_download.py (hunk 1 above)
Comment (johnlambert-argo) [reply integration_tests/verify_tbv_download.py:25]: Thanks, updated now.

File: integration_tests/verify_tbv_download.py (hunk 2 above)
Comment (johnlambert-argo) [reply integration_tests/verify_tbv_download.py:37]: Thanks, updated now.

**Pull request merged**